Common/Source/Devices/devLX16xx.cpp

```diff
@@ -356,12 +357,12 @@ bool DevLX16xx::LXWP0(PDeviceDescriptor_t d, const TCHAR* sentence, NMEA_INFO* i
     info->VarioAvailable = FALSE;
 
 
+
```

Comment:
Why ?

Reply:
ok, so you need to squash the 2 commit for have clean history...

Reply:
No the first rebase commit out commented out wind  by mistake.
And I re-enabled it with the rebase commit.

Reply:
cf. "squashing commit" : https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

Reply:
And how to do it in Github or Smartgit? I'm a bit afraid of the  git command since I lost some of my commits with it.

Reply:
easy : https://www.google.com/search?q=smartgit+interactive+rebase

Reply:
Does not work with SmartGit8 nor Online.